spec/operations/index.md

```diff
@@ -10,4 +10,4 @@ These OperationDefinitions have been defined for this implementation guide.
 * [Patient Export: export patient data from a FHIR server](../OperationDefinition-patient-export.html)
 * [Group Export: export data for groups of patients from a FHIR server](../OperationDefinition-group-export.html)
 
-
+We will define in the Bulk Data spec a cannonical URL that conforming servers can point to to declare conformance. Pointed to by `CapabilityStatement.instantiates`.
```

Comment:
This looks like text accidentally copied from the ballot disposition. It's an instruction for us (editors) to follow in updating the spec; not text for a reader to see. @ricky1uphealth Do you want to use this spot to document what the canonical URL is and how a server can claim conformance to it?

Reply:
@ricky1uphealth  update to this
To declare conformance with this IG a server should include the following URL in its own `CapabilityStatement.instantiates`: [http:.. capability stat url]()

Reply:
i addressed the capability statement and got it to link correctly in the hl7 rendered view.